defaults/main.yml

```diff
@@ -1,3 +1,6 @@
 ---
 keepalived_install: False
 keepalived_packages: "{{ __keepalived_packages }}"
+vrrp_script.weight_enabled: true
+vrrp_script.fall_enabled: true
+vrrp_script.rise_enabled : true
```

Comment:
I am firmly against adding such variables, it tends to clutter roles uselessly. A better approach is to remove the default value.

Reply:
I just removed them and removed the default value. Is it all right for you now?

Reply:
Yes, much better.